src/js/tabs/failsafe.js

```diff
@@ -297,9 +297,18 @@ failsafe.initialize = function (callback) {
 
         if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
             if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46)) {
-                $('input[name="gps_rescue_min_dth"]').attr("min", 10);
-            } else if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
-                $('input[name="gps_rescue_min_dth"]').attr("min", 20);
+                $('input[name="gps_rescue_min_dth"]').attr({"min": 10, "max": 30});
+                $('input[name="gps_rescue_ground_speed"]').attr({"min": 3.0, "max": 30.0, "step": 0.1});
+                $('input[name="gps_rescue_ascend_rate"]').attr({"min": 1.0, "max": 25.0, "step": 0.1});
```

Comment:
Please just change the step resolution in the html, so that all users benefit, regardless of their firmware level.
The previous 0.01m/s resolution for velocity settings was just crazy.

Reply:
Update - to see a value in Configurator without two decimal points, we need to set the input `toFixed(1)`, not `toFixed(2)`.
I recommend we do this, and change the html to use 0.1 steps, and change the default numbers in html to only one decimal place.
Being consistent throughout makes the code simpler.
And it won't adversely affect older versions, I expect that people using older versions will prefer to only have once decimal point to worry about.
Just changing the step in the javascript to 0.1 isn't enough.